### PR TITLE
Feature/create candidate

### PR DESCRIPTION
**Pull request by RemiFeydit**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,14 +25,16 @@
...
     "@nestjs/typeorm": "^8.0.3",
     "pg": "^8.7.3",
     "reflect-metadata": "^0.1.13",
     "typeorm": "0.2",
     "@nestjs/swagger": "^5.2.1",
     "rimraf": "^3.0.2",
     "rxjs": "^7.2.0",
-    "swagger-ui-express": "^4.3.0"
+    "swagger-ui-express": "^4.3.0",
+    "typeorm": "0.2"
```
Comment (saschasalles) [inline package.json]: typeorm  here why ?

File: src/app.module.ts [hunk 2]
```diff
@@ -1,28 +1,32 @@
...
 @Module({
   imports: [
     TypeOrmModule.forRoot({
       type: 'postgres',
       host: 'localhost',
       port: 5432,
-      username: '',
-      password: '',
+      username: 'notitou',
+      password: 'new',
```
Comment (saschasalles) [inline src/app.module.ts]: Just leave it free babe

File: src/candidate/entities/candidate.entity.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+} from 'typeorm';
+import { Tag } from './tag.entity';
+
+@Entity()
+export class Candidate {
+  @PrimaryGeneratedColumn()
+  id: string;
+
+  @Column()
+  url: string;
```
Comment (theoDELAS) [inline src/candidate/entities/candidate.entity.ts]: C'est quoi url ?

File: src/candidate/candidate.controller.spec.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import { Test, TestingModule } from '@nestjs/testing';
```
Comment (saschasalles) [inline src/candidate/candidate.controller.spec.ts]: Can remove this file ? This is not needed

File: src/candidate/candidate.service.spec.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+      providers: [CandidateService],
+    }).compile();
+
+    service = module.get<CandidateService>(CandidateService);
+  });
+
+  it('should be defined', () => {
+    expect(service).toBeDefined();
+  });
+});
```
Comment (saschasalles) [inline src/candidate/candidate.service.spec.ts]: Idem, can you remove this file ?

File: src/candidate/entities/candidate.entity.ts [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+} from 'typeorm';
+import { Tag } from './tag.entity';
+
+@Entity()
+export class Candidate {
+  @PrimaryGeneratedColumn()
+  id: string;
+
+  @Column()
+  profilePicUrl: string;
```
Comment (theoDELAS) [inline src/candidate/entities/candidate.entity.ts:16]: C'est quoi pic ?

**Pull request merged**